core/src/apps/mintlayer/sign_tx/signer.py

```diff
@@ -300,6 +300,60 @@ async def step1_process_inputs(
                 else:
                     raise DataError("Unknown account command")
                 self.tx_info.add_input(txi, None, nodes)
+            elif txi.order_command:
+                x = txi.order_command
+                nodes = []
+                for address in x.addresses:
+                    nodes.append(
+                        (
+                            self.keychain.derive(address.address_n),
+                            address.multisig_idx,
+                        )
+                    )
+                if x.fill:
```

Comment:
The logic is the same as for the v0 code (except for where "give" and "ask" values come from), can we factor it out?